File: readability/readability.c
#include <cs50.h>
#include <stdio.h>
#include <string.h>
#include <ctype.h>
#include <math.h>

int count_letters(string texts);
int count_words(string texts);
int count_sentences(string texts);


int main(void)
{
    string texts = get_string("text: ");

    int letterc = count_letters(texts);
    int wordc = count_words(texts);
    int sentencec = count_sentences(texts);


    //printf("%s \n", texts);
    //printf("letters : %i  \n", letterc);
    //printf("words : %i  \n", wordc);
    //printf("sentences : %i  \n", sentencec);

    float L = 100 * letterc / wordc; // L = average letter in 100 words
    float S = 100 * sentencec / wordc; // S = average sentence in 100 words
    int index = round(0.0588 * L - 0.296 * S - 15.800);

    if (index >= 1 && index <= 16)
    {
         printf("Grade %i\n", index);
    }
    else if (index < 1)
    {
        printf("Before Grade 1\n");
    }
    else
    printf("Grade 16+\n");


}





   int count_letters(string texts)
    {
        int letters = 0;
        for (int i=0; i < strlen(texts); i++)
        {
            if (texts[i] > 96 && texts[i] < 122)
            {
                 letters = letters + 1;
            }
            else if (texts[i] > 64 && texts[i] < 91)
            {
                  letters = letters + 1;
            }
            else letters = letters + 0;

        }
        return letters;
    }

    int count_words(string texts)
    {
        int words = 1;
        for (int w = 0; w < strlen(texts); w++)
        {
            if (isblank(texts[w]))
            {
                words = words + 1;
            }
            else words = words + 0;
        }
        return words;
    }


    int count_sentences(string texts)
    {
        int sentences = 0;
        for (int s = 0; s < strlen(texts); s++)
        {
            if (texts[s] == 46 || texts[s] == 33 || texts[s] == 63)
            {
                sentences = sentences + 1;
            }
        }
        return sentences;
    }

File: sentimental-readability/readability.py
# TODO
import cs50

def main():
    texts = cs50.get_string("text:")
    lettercount = count_letter(texts)
    wordcount = count_word(texts)
    sentencecount = count_sentence(texts)

    L = float(100 * lettercount / wordcount)
    S = float(100 * sentencecount / wordcount)
    index = round (0.0588 * L - 0.296 * S - 15.800)

    print_grade(index)

def count_letter(text):
    letters = 0
    for z in range(len(text)):
        letterz = ord(text[z])
        if (letterz > 96 and letterz < 122) or (letterz > 64 and letterz < 91):
            letters = letters + 1
    return letters

def count_word(text):
    words = 1
    for x in range(len(text)):
        if (text[x] == " "):
            words = words + 1
    return words

def count_sentence(text):
    sentences = 0
    for y in range (len(text)):
        if (text[y] == "." or text[y] == "!" or text[y] == "?"):
            sentences = sentences + 1
    return sentences

def print_grade(index):
    if index >= 1 and index <= 16:
        print("Grade", index)
    elif index < 1:
        print("Before Grade 1\n")
    else:
    	print("Grade 16+\n")


if __name__ == "__main__":
    main()